oat-sa/extension-tao-itemqti
# Feature/tao 4554 math support in gapmatch

https://oat-sa.atlassian.net/browse/TAO-4554

How to test:
- create a gapMatch interaction
- add a choice or use the default one
- by clicking on the ckEditor toolbar, you should now be able to add a Math Element into the gap choice.

Alternative choice creation method:
- in the item body, insert a Math expression
- select the math expression
- click the magic wand on the right
- the new choice should contain the math expression
